src/adapters/db.ts

```diff
@@ -2,17 +2,43 @@ import SQL from 'sql-template-strings'
 import { AppComponents } from '../types'
 
 export type IDatabaseComponent = {
-  getCommunityMembers: (communityId: string) => Promise<string[]>
+  getCommunityMembers: (
+    communityId: string,
+    options?: {
+      include?: string[]
+      exclude?: string[]
+    }
+  ) => Promise<string[]>
 }
 
-export async function createDBComponent(components: Pick<AppComponents, 'pg' | 'logs'>): Promise<IDatabaseComponent> {
-  const { pg, logs } = components
-  const _logger = logs.getLogger('database')
+export async function createDBComponent(components: Pick<AppComponents, 'pg'>): Promise<IDatabaseComponent> {
+  const { pg } = components
 
-  const getCommunityMembers = async (communityId: string) => {
-    const result = await pg.query(
-      SQL`SELECT LOWER(address) as address FROM community_members WHERE community_id = ${communityId}`
-    )
+  const getCommunityMembers = async (
+    communityId: string,
+    options?: {
+      include?: string[]
+      exclude?: string[]
+    }
+  ) => {
+    const includeAddresses = options?.include?.map((address) => `'${address}'`)
+    const excludeAddresses = options?.exclude?.map((address) => `'${address}'`)
```

Comment:
```suggestion
    const includeAddresses = options?.include?.map((address) => `'${address.toLowerCase()}'`)
    const excludeAddresses = options?.exclude?.map((address) => `'${address}.toLowerCase()'`)
```

or using the LOWER from Postgre directly 👼

Reply:
In the social service, we discovered that it is not very efficient to do the lower in pg, so I would rather keep it like this. WDYT?